### PR TITLE
Bug 1307201, replaced publicMasterURL with masterPublicURL

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1307201

### CODE REVIEW

File: install_config/certificate_customization.adoc [hunk 1]
```diff
@@ -36,7 +36,8 @@ A default certificate must be configured in the `*servingInfo.certFile*` and
 [NOTE]
 ====
 The `*namedCertificates*` section should only be configured for the host name
-associated with the `*publicMasterURL*`. Using a custom serving certificate for
+associated with the `*masterPublicURL*`, `*assetConfig.publicURL*`, and
+`*oauthConfig.assetPublicURL*`` settings. Using a custom serving certificate for
```
Comment (adellape) [inline install_config/certificate_customization.adoc]: Extra backtick here.

**Comment by adellape** [comment]
@ahardin-rh This topic doesn't exist in the enterprise-3.0 branch, and doesn't pertain to online since it's an installation topic, so both of those labels can be dropped.

**Comment by ahardin-rh** [comment]
@adellape Fixed. Thanks!

**Comment by ahardin-rh** [comment]
Approval received from reporter, James Radtke.

**Comment by ahardin-rh** [comment]
**revhistory note:**

(Installation and Configuration)

    In the link:../install_config/certificate_customization.html#configuring-custom-certificates[Configuring Custom Certificates] section, replaced publicMasterURL with masterPublicURL.

**Pull request merged**